modules/nf-core/picard/bedtointervallist/main.nf

```diff
@@ -0,0 +1,57 @@
+process PICARD_BEDTOINTERVALLIST {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda (params.enable_conda ? "bioconda::picard=2.27.4" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/picard:2.27.4--hdfd78af_0' :
+        'quay.io/biocontainers/picard:2.27.4--hdfd78af_0' }"
+
+    input:
+    tuple val(meta) , path(bed)
+    tuple val(meta2), path(dict)
+
+    output:
+    tuple val(meta), path('*.interval_list'), emit: interval_list
+    path  "versions.yml"                    , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    def avail_mem = 3
+    if (!task.memory) {
+        log.info '[Picard BedToIntervalList] Available memory not known - defaulting to 3GB. Specify process memory requirements to change this.'
+    } else {
+        avail_mem = task.memory.giga
+    }
+    """
+    picard \\
+        -Xmx${avail_mem}g \\
```

Comment:
Missing optional input of `arguments_file`, `references_file`, `tmp_dir` (although not sure about htis one however I've had problems with picard on some clusters because of this so might be nice),

Reply:
Why would you add an arguments file if you could use configs in Nextflow? It's probably better to do the arguments all in the same place? (I've never added it to GATK tools for that reason at least)

Reply:
:shrug: why would you, I don't know. But we shouldn't assume what people want to do ;)

Reply:
Yeah true, fair point there :p

Reply:
👎🏻  on the arguments file IMO, it adds an extra input and is almost never used in practice, nor is it accounted for in the bulk of the picard/gatk modules

Reply:
`--TMP_DIR` is set to `$PWD` in accordance to the rest of the modules where it's defined

Reply:
`-R` is one of the cruft params added by default in GATK, not sure about Picard. It's not used here anyways, since the tool works on the `.dict` file

Reply:
@jfy133, opinions?

Reply:
> 👎🏻 on the arguments file IMO, it adds an extra input and is almost never used in practice, nor is it accounted for in the bulk of the picard/gatk modules

Then it's done wrong ;) (I blame Sarek people). I would stick with this one. Remember the modules can be used outside of nf-core pipelines so some people _may_ want to use it.

For `--TMP_DIR` that's Ok then - I had problems in another pipeline with this where PICARD would try and use a clusters `/tmp` (which wasn't allowed) and changing this was really difficult but if it's just $PWD fine with me.

If you're sure it's not used here it's fine to exclude it.

Reply:
Thanks! I'll add the arguments file while shaking my fist a the sarek people 😄 ✊🏻

Reply:
Done! Ready for (another) review!